packages/dd-trace/src/profiling/util.js

```diff
@@ -57,8 +26,5 @@ function parallel (tasks, callback) {
 }
 
 module.exports = {
-  maybeRequire,
-  parallel,
-  eachSeries,
-  eachOfSeries
+  parallel
```

Comment:
Parallel could also be removed if promises are used in every place where these helpers were used.